src/domain/app/notFound/NotFound.tsx

```diff
@@ -1,15 +1,26 @@
 import React, { FunctionComponent } from 'react';
 import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
 
+import styles from './notFound.module.scss';
 import Container from '../layout/Container';
+import Icon from '../../../common/components/icon/Icon';
+import adultFace from '../../../assets/icons/svg/adultFace.svg';
+import Button from '../../../common/components/button/Button';
 
 const NotFound: FunctionComponent = () => {
   const { t } = useTranslation();
-
+  const history = useHistory();
   return (
     <Container>
-      <div className="not-found">
+      <div className={styles.notFound}>
+        <h2>404</h2>
+        <Icon src={adultFace} className={styles.icon} alt="Sad adult face" />
         <p>{t('notFound.text')}</p>
+
+        <Button onClick={() => history.push('/')}>
```

Comment:
This should probably be a `<Link>`

Reply:
I think it should be Button with function of a link, as the design is a button :D

Reply:
No.

Reply:
![Screenshot 2020-01-15 at 13 22 45](https://user-images.githubusercontent.com/9389531/72430099-285e9580-379a-11ea-9d6c-c166b4d1634d.png)

Reply:
You can style an anchor-element to look like a button. It's a link, not a button.

Reply:
okay :sad: